src/main/kotlin/no/nav/syfo/melding/database/MeldingQuery.kt

```diff
@@ -153,6 +153,24 @@ private fun Connection.createMelding(
     return id
 }
 
+const val queryGetMeldingerForConversation =
+    """
+        SELECT *
+        FROM MELDING
+        WHERE conversation_ref = ?
+    """
+
+fun DatabaseInterface.getMeldingerForConversation(
```

Comment:
Er det bedre med `meldingerInConversation`? Med `for` høres det litt ut som at det er samtalen som har bedt om meldinger 🤔